docs/source/api/index.rst

```diff
@@ -22,7 +22,7 @@ The Traffic Ops API provides programmatic access to read and write Traffic Contr
 
 How to Read this Documentation
 ==============================
-Each endpoint for each version is on its own page, titled with the request path. The request paths shown on each endpoint's page are - unless otherwise noted - only usable by being appended to the request path prefix ``/api/<version>/`` where ``<version>`` is the API version being requested. The API versions officially supported as of the time of this writing are 1.1, 1.2, 1.3, 1.4, 1.5, 2.0. All endpoints are documented as though they were being used in version 1.5 in the version 1 documentation and version 2.0 in the version 2 documentation. If an endpoint or request method of an endpoint is only available after a specific version, that will be noted next to the method or endpoint name. If changes were made to the structure of an endpoint's input or output, the version number and nature of the change will be noted.
+Each endpoint for each version is on its own page, titled with the request path. The request paths shown on each endpoint's page are - unless otherwise noted - only usable by being appended to the request path prefix ``/api/<version>/`` where ``<version>`` is the API version being requested. The API versions officially supported as of the time of this writing are 3.0, 4.0. All endpoints are documented as though they were being used in version 3.0 in the version 3 documentation and version 4.0 in the version 4 documentation. If an endpoint or request method of an endpoint is only available after a specific version, that will be noted next to the method or endpoint name. If changes were made to the structure of an endpoint's input or output, the version number and nature of the change will be noted.
```

Comment:
> 3.0, 4.0

This is missing an "and" and also the 3.1 API version.

Also the endpoints are documented as though they were being used in version 3.1 in the version 3 documentation, not 3.0. Well, quite a lot of them are actually 3.0, but only where there's no difference between 3.0 and 3.1.